cpp/include/cudf/utilities/type_dispatcher.hpp

```diff
@@ -49,11 +49,11 @@ namespace CUDF_EXPORT cudf {
  * return cudf::type_to_id<int32_t>();        // Returns INT32
  * ```
  *
- * @tparam T The type to map to a `cudf::type_id`
+ * @tparam T The non-cv type to map to a `cudf::type_id`
  * @return The `cudf::type_id` corresponding to the specified type
  */
 template <typename T>
-CUDF_HOST_DEVICE inline constexpr type_id type_to_id()
+CUDF_HOST_DEVICE inline constexpr type_id type_to_id_helper()
```

Comment:
What do you think of the name `base_type_to_id`?

Reply:
Sure!